Snapshot

joemontalvo opened the pull request.

joemontalvo committed:
Cleaned up Code + added removal safety button

joemontalvo committed:
Cleaned up Code + added removal safety button

joemontalvo committed:
Merge remote-tracking branch 'origin/snapshot' into snapshot

joemontalvo committed:
Merge branch 'PavlAvstin:snapshot' into snapshot

joemontalvo committed:
Finalized HandleSlashCommands:
- documented all of the functions and what they are supposed to do.
- added parsing to the add and remove pair commands to check if the input is valid or not before sending a request to the database.*

*only certain emojis are supported because of this check. The library I used to check the input has a limited set of emojis that it works for. It covers most of the emojis but not all of them.

joemontalvo committed:
Got rid of the remove pair command and switched it for remove reaction.
this simplified the reactions and changed a few function names etc.
I fixed buttons not disappearing after being clicked. although im worried there may be an issue with who is allowed to press the button and if it is public for everyone to use / interact with.

fixed the /remove dictionary slash command so that it removes all reactions from the db including ones that have been used.

kyleyannelli reviewed (changes requested):
looks okay other than the .envexample being deleted. Can you roll that back then do another pr?

joemontalvo committed:
Got rid of the remove pair command and switched it for remove reaction.
this simplified the reactions and changed a few function names etc.
I fixed buttons not disappearing after being clicked. although im worried there may be an issue with who is allowed to press the button and if it is public for everyone to use / interact with.

fixed the /remove dictionary slash command so that it removes all reactions from the db including ones that have been used.
